client/stylesheets/custom.less

```diff
@@ -16,6 +16,12 @@ body {
   color: rgb(51, 53, 49) !important;
 }
 
+.form-group.required .control-label:before {
```

Comment:
The floating label uses the **:after** thing already, so messes it up :disappointed: 

That was the way I found so far to do this, feel free to try something better, I think that the asterisk would be better in the right side of the label